cmd/jaeger/internal/extension/remotesampling/extension_test.go

```diff
@@ -84,7 +83,7 @@ func makeRemoteSamplingExtension(t *testing.T, cfg component.Config) component.H
 func TestStartFileBasedProvider(t *testing.T) {
 	factory := NewFactory()
 	cfg := factory.CreateDefaultConfig().(*Config)
-	cfg.File.Path = filepath.Join("..", "..", "..", "sampling-strategies.json")
+	cfg.File = nil
```

Comment:
why do you need to change this test?

Reply:
Apologies for the delay.
I was trying to make the the tests independant of the sampling-strategies file. And so I was giving a null value to the input input to prompt default behavior. The test was working either ways(input was file based in both cases finally).
Still, I have removed the changes now.